programs/delayed_node/main.cpp

```diff
@@ -70,14 +70,14 @@ int main(int argc, char** argv) {
 
       bpo::variables_map options;
 
-      auto delayed_plug = node.register_plugin<delayed_node::delayed_node_plugin>();
-      auto history_plug = node.register_plugin<account_history::account_history_plugin>();
-      auto market_history_plug = node.register_plugin<market_history::market_history_plugin>();
+      auto delayed_plug = node->register_plugin<delayed_node::delayed_node_plugin>();
```

Comment:
Perhaps `auto delayed_plug = node->register_plugin<delayed_node::delayed_node_plugin>(true);`?

Reply:
make sense in this context, added. thanks.